### PR TITLE
docs: added setup options to docs

### PR DESCRIPTION
**Pull request by stebenz**

### CODE REVIEW

File: docs/docs/concepts/setup.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            <li>Changes for specific environments</li>
+            <li>Performance testing</li>
+            <li>Production deployment</li>
+            <li>Application-side coding, configuration, or tuning</li>
+            <li>Changes or configuration on assets used in ZITADEL</li>
+            <li>Setting up or maintaining backup</li>
+        </ul>
+        <p>Prerequisites</p>
+        <ul>
+            <li>Possibility to run ORBOS with one of the implemented Providers</li>
```
Comment (mffap) [inline docs/docs/concepts/setup.md:82]:
```suggestion
            <li>Possibility to run ORBOS with one of the [implemented providers](https://github.com/caos/orbos/blob/master/docs/orbiter/providers.md)</li>
```

File: docs/docs/concepts/setup.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+        <p>Time Estimate</p>
+        <ul>
+            <li>8 hours</li>
+        </ul>
+    </TabItem>
+    <TabItem value="orbos">
+        <p>In Scope</p>
+        <ul>
+            <li>Check prerequisites and architecture</li>
+            <li>Setup of Kubernetes cluster with ORBOS</li>
```
Comment (mffap) [inline docs/docs/concepts/setup.md:62]:
```suggestion
            <li>Setup of Kubernetes cluster with [ORBOS](https://github.com/caos/orbos)</li>
```

File: docs/docs/concepts/setup.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            <li>Changes for specific environments</li>
+            <li>Performance testing</li>
+            <li>Application-side coding, configuration, or tuning</li>
+            <li>Changes or configuration on assets used in ZITADEL</li>
+            <li>Setting up or maintaining backup</li>
+        </ul>
+        <p>Prerequisites</p>
+        <ul>
+            <li>Access to the Kubernetes cluster or physical/virtual nodes</li>
+            <li>Environment to test and check should not be in productive use</li>
```
Comment (mffap) [inline docs/docs/concepts/setup.md:122]:
```suggestion
            <li>Environment to test and check (should not be in productive use)</li>
```

File: docs/docs/concepts/setup.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            <li>Access to used configuration</li>
```
Comment (mffap) [inline docs/docs/concepts/setup.md:123]: Additionally maybe?

- Access to monitoring
- Access to relevant system and application logs

File: docs/docs/concepts/setup.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+            <li>Integration into internal monitoring and alerting</li>
+            <li>Changes for specific environments</li>
+            <li>Performance testing</li>
+            <li>Application-side coding, configuration, or tuning</li>
+            <li>Changes or configuration on assets used in ZITADEL</li>
+            <li>Setting up or maintaining backup</li>
+        </ul>
+        <p>Prerequisites</p>
+        <ul>
+            <li>Access to the Kubernetes cluster or physical/virtual nodes</li>
```
Comment (mffap) [inline docs/docs/concepts/setup.md:121]:
```suggestion
            <li>Access to relevant Kubernetes cluster or physical/virtual nodes</li>
```

Comment (mffap) [inline docs/docs/concepts/setup.md:121]: Additionally: Do you need access to asset storage, backup storage, maybe database cluster/node?

**Review by mffap — CHANGES_REQUESTED** [review]
Looks very good and easy to understand! 
Some questions and suggestions.

**Review by mffap — DISMISSED** [review]

File: docs/sidebars.js [hunk 6]
```diff
@@ -71,6 +71,16 @@ module.exports = {
             'guides/installation/managed-dedicated-instance'
           ],
         },
+        {
+          type: 'category',
+          label: 'Setup by CAOS',
```
Comment (fforootd) [inline docs/sidebars.js:76]:
```suggestion
          label: 'CAOS Service Packages',
```

File: docs/docs/guides/installation/setup-orbos.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+title: ORBOS and ZITADEL setup by CAOS
```
Comment (fforootd) [inline docs/docs/guides/installation/setup-orbos.md:2]:
```suggestion
title: ZITADEL setup with ORBOS
```

File: docs/docs/guides/installation/setup.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+---
+title: ZITADEL setup by CAOS
```
Comment (fforootd) [inline docs/docs/guides/installation/setup.md:2]:
```suggestion
title: ZITADEL setup
```

**Review by mffap — APPROVED** [review]

**Pull request merged**